bibliography.bib

```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
+    journal      = AI,
+    series       = {},
+    volume       = {297},
+    number       = {},
+    year         = {2021},
+    pages        = {103--500},
+    doi          = {10.1016/j.artint.2021.103500},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BuşBabDeS2010imasa,
+    title        = {Multi-agent reinforcement learning: an overview},
+    author       = Busoniu_L # and # Babuska_R # and # DeSchutter_B,
+    journal      = IMASA,
+    series       = {},
+    volume       = {297},
+    number       = {},
+    year         = {2010},
+    pages        = {183--221},
+    doi          = {10.1007/978-3-642-14435-6_7},
+    note         = {},
+    annote       = {},
+}
+
+@Article{WatDay1992ml,
+    title        = {Q-learning},
+    author       = Watkins_C # and # Dayan_P,
+    journal      = ML,
+    series       = {},
+    volume       = {8},
+    number       = {},
+    year         = {1992},
+    pages        = {279-–292},
+    doi          = {10.1007/BF00992698},
+    note         = {},
+    annote       = {},
+}
+
+@Article{MniKavSil-etal2015natu,
+    title        = {Human-level control through deep reinforcement learning},
+    author       = Mnih_V # and # Kavukcuoglu_K # and # Silver_D # and # Rusu_A # and # Veness_J # and # Bellemare_MG # and # Graves_A # and # Riedmiller_M # and # Fidjeland_AK # and # Ostrovski_G,
+    journal      = NATU,
+    series       = {},
+    volume       = {518},
+    number       = {7540},
+    year         = {2015},
+    pages        = {529--533},
+    doi          = {10.1038/nature14236},
+    note         = {},
+    annote       = {},
+}
+
+@Article{Nob2006natubiot,
+    title        = {What is a support vector machine?},
+    author       = Noble_WS,
+    journal      = NATUBIOT,
+    series       = {},
+    volume       = {24},
+    number       = {12},
+    year         = {2006},
+    pages        = {1565--1567},
+    doi          = {10.1038/nbt1206-1565},
+    note         = {},
+    annote       = {},
+}
+
+
+@Article{HorStiWhi1989nn,
+    title        = {Multilayer feedforward networks are universal approximators},
+    author       = Hornik_K # and # Stinchcombe_M # and # White_H,
+    journal      = NN,
+    series       = {},
+    volume       = {2},
+    number       = {5},
+    year         = {1989},
+    pages        = {359--366},
+    doi          = {10.1016/0893-6080(89)90020-8},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BelHsuMa-etal2019pnas,
+    title        = {Reconciling modern machine-learning practice and the classical bias--variance trade-off},
+    author       = Belkin_M # and # Hsu_D # and # Ma_S # and # Mandal_S,
+    journal      = PNAS,
+    series       = {},
+    volume       = {116},
+    number       = {32},
+    year         = {2019},
+    pages        = {15849--15854},
+    doi          = {10.1073/pnas.1903070116},
+    note         = {},
+    annote       = {},
+}
+
+@Book {HasLigFraBir2019book,
```

Comment:
this is duplicated
https://github.com/demiurge-project/demiurge-bib/blob/5c51777aae7c3ef2292759a85850f7395276e218/bibliography.bib#L260